### DISCUSSION TITLE
"rx_counter" not a zero in 1 of 75 devices in coordinator_backup.json

### DISCUSSION THREAD

**Discussion by mrmaximas** [Koenkk/zigbee2mqtt#21156] [General]
Zigbee2MQTT version
1.35.2-dev commit: [da1d0d81](https://github.com/Koenkk/zigbee2mqtt/commit/da1d0d81)
Coordinator type
zStack3x0
Coordinator revision
20221226
Coordinator IEEE Address
0x00124b002a1aa3c4
Frontend version
0.6.152
Zigbee-herdsman-converters version
18.26.0
Zigbee-herdsman version
0.33.5

Analysing the coordinator's backup I found that only 1 of my 75 active devices, such as https://www.zigbee2mqtt.io/devices/LED1837R5.html, has a received packet count that is not zero.
{
      "nwk_address": "44bd",
      "ieee_address": "70ac08fffeb366f8",
      "is_child": false,
      "link_key": {
        "key": "censored",
        "rx_counter": 57345,
        "tx_counter": 4041
      }
    },
    
All other devices have a value of zero.
For example
    {
      "nwk_address": "46c2",
      "ieee_address": "54ef441000947c46",
      "is_child": false,
      "link_key": {
        "key": "censored",
        "rx_counter": 0,
        "tx_counter": 2805
      }
    },
    
   What could it be?

**Comment by thargy** [comment]
I'd love to understand the meanings of `rx_counter`/`tx_counter` too, if anyone can illuminate me. I also have a lot of counters that don't change for days, then others, with an indiscernible pattern, that have very different, and often large, values.
